exec/java-exec/src/test/java/org/apache/drill/test/TestGracefulShutdown.java

```diff
@@ -73,31 +73,39 @@ private static void enableDrillPortHunting(ClusterFixtureBuilder builder) {
     builder.configBuilder.put(ExecConstants.DRILL_PORT_HUNT, true);
     builder.configBuilder.put(ExecConstants.GRACE_PERIOD, 500);
     builder.configBuilder.put(ExecConstants.ALLOW_LOOPBACK_ADDRESS_BINDING, true);
+
+    setTestDirectories(builder);
+  }
+
+  private static void setTestDirectories(ClusterFixtureBuilder builder) {
+    builder.configBuilder.put(ExecConstants.DRILL_TMP_DIR, dirTestWatcher.getTmpDir().getAbsolutePath());
+    builder.configBuilder.put(ExecConstants.SYS_STORE_PROVIDER_LOCAL_PATH, dirTestWatcher.getStoreDir().getAbsolutePath());
```

Comment:
Can this be done in `ClusterFixture` or its builder so we use a consistent set of directories everywhere? I've been burned by these being a bit ill-defined.